turbopack/crates/turbopack-ecmascript-runtime/js/src/nodejs/runtime.ts

```diff
@@ -93,6 +93,13 @@ function loadChunk(chunkData: ChunkData, source?: SourceInfo): void {
 }
 
 const loadedChunks = new Set<ChunkPath>()
+const unsupportedLoadChunk = Promise.resolve(undefined)
+const loadedChunk = Promise.resolve(undefined)
+const chunkCache = new Map<ChunkPath, Promise<any> | null>()
```

Comment:
I understand you want to avoid holding onto extra promises, but it does seem a bit confusing (evidenced by Vade's (the AI) wrong comment) that `null` is used to store the success state.

Why not just store `loadedChunk` as the map's value instead of `null` after success?

Reply:
Makes sense. I'll do that in a follow-up